src/spaces/graph.jl

```diff
@@ -23,33 +23,42 @@ end
     nv(model::ABM)
 Return the number of nodes (vertices) in the `model` space.
 """
-LightGraphs.nv(abm::ABM{<:Any, <:GraphSpace}) = LightGraphs.nv(abm.space.graph)
+LightGraphs.nv(abm::ABM{<:Any,<:GraphSpace}) = LightGraphs.nv(abm.space.graph)
 
 """
     ne(model::ABM)
 Return the number of edges in the `model` space.
 """
-LightGraphs.ne(abm::ABM{<:Any, <:GraphSpace}) = LightGraphs.ne(abm.space.graph)
+LightGraphs.ne(abm::ABM{<:Any,<:GraphSpace}) = LightGraphs.ne(abm.space.graph)
 
-function Base.show(io::IO, abm::GraphSpace)
-    s = "$(nameof(typeof(abm))) with $(nv(abm)) nodes and $(ne(abm)) edges"
-    print(io, s)
+function Base.show(io::IO, space::GraphSpace)
+    print(
+        io,
+        "$(nameof(typeof(space))) with $(nv(space.graph)) nodes and $(ne(space.graph)) edges",
+    )
 end
 
 
 #######################################################################################
 # Agents.jl space API
 #######################################################################################
-random_position(model::ABM{<:AbstractAgent, <:GraphSpace}) = rand(1:nv(model))
+random_position(model::ABM{<:AbstractAgent,<:GraphSpace}) = rand(1:nv(model))
 
-function remove_agent_from_space!(agent::A, model::ABM{A,<:GraphSpace}) where {A<:AbstractAgent}
+function remove_agent_from_space!(
+    agent::A,
+    model::ABM{A,<:GraphSpace},
+) where {A<:AbstractAgent}
```

Comment:
Can you configure your editor / linter to ident these one level? it is intuitive to see a code block closing and the next to start at a higher identation level.